spec/devise/jwt/mapping_inspector_spec.rb

```diff
@@ -15,6 +15,10 @@
     described_class.new(:jwt_with_blacklist_user)
   end
 
+  let(:jwt_with_whitelist_inspector) do
+    described_class.new(:jwt_with_whitelist_user)
+  end
+
```

Comment:
There is no need to declare this `let` variable because it is not used in the tests